WIP - MTA-4637: Pattern matching docs need urgent improvements (#92)
migtools/mta-documentation: docs/topics/yaml-chaining-condition-variables.adoc

// Module included in the following assemblies:
//
// * docs/rules-development-guide/master.adoc

:_mod-docs-content-type: REFERENCE
[id="yaml-chaining-condition-variables_{context}"]
= Chaining Condition Variables

> It is also possible to use the output of one condition as the input for filtering another one in the `and` and `or` conditions. This is called *condition chaining*.

Reviewer: ```suggestion
You can use the output of one condition as the input for filtering another one in the `and` and `or` conditions. This is called *condition chaining*.
```
